internal/parse/parser_test.go

```diff
@@ -29,113 +29,125 @@ type M map[string]any
 
 func TestRound(t *testing.T) {
 	var tests = []struct {
+		name           string
```

Comment:
The examples below look more like a "summary" than a "name". I frequently do something similar as well,btw... it's very convenient for the organization, so thanks for doing it.

Reply:
Changed and rebased :)